include/cutlass/gemm/collective/builders/xe_mma_builder.inl

```diff
@@ -41,6 +41,81 @@ namespace cutlass::gemm::collective {
   // Intel PVC 3 stage pipeline, using prefetch
   // Also the auto builder
 
+template<typename MMAAtom, typename T_m, typename T_n>
+constexpr auto get_n_atoms(T_m tile_m, T_n tile_n){
```

Comment:
```suggestion
constexpr auto get_atoms_dims(T_m tile_m, T_n tile_n){
```
the n made me think of the dimension

Reply:
while get_atoms_dims makes me think that this returns dimensions of one atom. I think `get_num_atoms` should be unambiguous.